sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -302,6 +302,13 @@ function buildAuthorizationRule(value: any): AuthorizationRule {
     primaryKey: value["PrimaryKey"],
     secondaryKey: value["SecondaryKey"]
   };
+
+  if (
+    authorizationRule.rights.accessRights &&
+    !Array.isArray(authorizationRule.rights.accessRights)
```

Comment:
I just noticed line 299
`accessRights: value["Rights"]["AccessRights"]`
This could cause an exception if `value["Rights"] is undefined. Since you're touching this function already can you fix that as well?

Reply:
updated

Reply:
Thank you!